LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2013 Despo Pentara
```

Comment:
Ok. I need something to think and discuss this with the organisers so we can figure out what to do. We (codebar) should also change our license so maybe we can figure out a way that is more suitable and enables attribution.

In the meantime, any tips/suggestions are welcome.